packages/kotti-ui/source/next/kotti-field-date/styles.scss

```diff
@@ -159,6 +159,38 @@ $year-font-size: 1rem;
 				}
 			}
 		}
+
+		.in-range {
+			background-color: var(--interactive-04);
```

Comment:
line 164 to 185 should be moved to their concerned part, under td: then add .in-range
the rest should be removed as explained below

thank you

Reply:
Not how I want structure my code

Reply:
@carsoli moved it, still appears to work.